content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -32,6 +32,8 @@ which are configured in the API.
 Admission controllers may be "validating", "mutating", or both. Mutating
 controllers may modify the objects they admit; validating controllers may not.
 
+Admission controllers have limited scopes. They only act on requests to create, delete, modify or connect (proxy), but not read.
```

Comment:
How about making it "connect to (proxy)" rather than just "connect (proxy)"?

Reply:
done!